account_reporting_weight/reports/account_invoice_report.py

```diff
@@ -13,12 +13,13 @@ class AccountInvoiceReport(models.Model):
     def _select(self):
         select_str = super()._select()
         select_str += """
-            , SUM(product.weight * (
-                CASE
-                WHEN move.type IN ('in_invoice','out_refund','in_receipt') THEN -1
-                ELSE 1 END
-            ) * line.quantity
-            / uom_line.factor * uom_template.factor)
-            as weight
+            , COALESCE(
```

Comment:
Why there's not `SUM` now and why `COALESCE` is needed?

Reply:
`SUM` required adding `GROUP BY`.
I have added `COALESCE` to not show NULL if, for example, a line has no product.

I have functionally tested with different use cases and the weight value is correct.

Is there any other way to make the change?